### DISCUSSION TITLE
Bosch Condens 5300i ud MQTT

### DISCUSSION THREAD

**Discussion by 3aau** [emsesp/EMS-ESP32#1359] [Q&A]
moin,
ich habe hier eine Bosch Condens 5300i + ct200.
erkannt wird sie als Bosch Condens 5100i.
verbindung zum ems-esp ist stabil und beide geräte werden zu 100% eingelesen.
ich möchte die daten gern mittels MQTT erfassen.
die ct200 liefert aber keine verwertbare daten. hier hätte ich gern die
aktuelle raumtemperatur ausgelesen. 
bei der 5300i felhlen mir in den daten die 
1.) aktuelle raumtemperatur
2.) aktuelle aussenteperatur
3.) aktueller gasverbrauch boiler
4.) aktueller gasverbrauch ww
5.) aktuellle rücklauftemperatur
hat da wer erfahrung mit?

moin,
I have here a Bosch Condens 5300i + ct200.
it is recognized as Bosch Condens 5100i.
connection to the ems-esp is stable and both devices are read 100%.
I would like to collect the data via MQTT.
but the ct200 does not provide usable data. here I would like to have the
read out the current room temperature. 
with the 5300i felhlen me in the data the 
1.) current room temperature
2.) current outside temperature
3.) current gas consumption boiler
4.) current gas consumption hot water
5.) current return temperature
does anyone have experience with this?

danke.

**Comment by proddy** [comment]
CT200 isn't fully EMS compliant. It sends all its data into the cloud directly and doesn't broadcast on the EMS bus. See https://emsesp.github.io/docs/Thermostats/?query=easy#list-of-thermostats

However, the current room temp and selected room temp should be available, on telegram 0x0A. 

For boiler consumption, we'll need to look into where the data is for the 5300i so will need your help to locate these messages. Also need some support info to help you further. See https://emsesp.github.io/docs/Support/

**Reply by MichaelDvP** [reply]
> However, the current room temp and selected room temp should be available, on telegram 0x0A.

This was last tested a long time ago (v1.9) and works there. In #205 the 0x0A shows only zeros. Maybe Bosch has added new telegrams in newer software versions. 
@3aau could you make a full log of some minutes, save it to file and attach the file here (**attach**, not paste the whole text). And also the support info from ems-esp to see the device info of the CT200 and telegrams.

**Comment by bbqkees** [comment]
With the TC100 the current room and setpoint are available, with the CT200 nothing is available.

**Reply by proddy** [reply]
let's see what the telegram log shows. if there is no support for the CT200 we'll remove it from the code.

**Comment by 3aau** [comment]
hier das logfile.
[ems-esp-log0001.txt](https://github.com/emsesp/EMS-ESP32/files/13225188/ems-esp-log0001.txt)

ich hoffe es hilft.

**Reply by proddy** [reply]
thanks. Michael is right, still all zero's in telegram 0x0A which means no selected or current room temperatures from the thermostat.

**Comment by 3aau** [comment]
moin,
gibt es irgendwas neues zu der software?
kann ich noch irgendwie behilflich sein?
